Python/sysmodule.c

```diff
@@ -1682,21 +1691,22 @@ sys_getwindowsversion_impl(PyObject *module)
         );
     }
 
-    if (realVersion) {
-        PyStructSequence_SET_ITEM(version, pos++, realVersion);
+    if (!realVersion) {
+        goto error;
     }
```

Comment:
It is no longer needed.

Reply:
I think it is needed, because `realVersion` is set to 
```c
        Py_BuildValue("(kkk)",
            ver.dwMajorVersion,
            ver.dwMinorVersion,
            ver.dwBuildNumber
        );
```

which can be `NULL`:

```
If there is an error in the format string, the :exc:`SystemError` exception is
   set and ``NULL`` returned.
```

So, if it is `NULL` we go to `error`.

Reply:
But SET_VERSION_INFO below checks for NULL.

Reply:
Indeed 👍 
Thank you1